metadata/la.json

```diff
@@ -8,6 +8,9 @@
 	"locations": [
```

Comment:
I think Cameron was worried about breaking the ZoneDB API / response format for other clients (besides us) that might be using it, if we change the structure of the `locations` array.